migrations/20200614165804_add_fn_ln_to_application.sql

```diff
@@ -0,0 +1,19 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+ALTER TABLE bm7_applications
```

Comment:
This should be fine for now, but as we move towards bm8 we'll probably want to make a bm8_applications table

Reply:
Can we use a general name like bm_applications?

Reply:
Yeah that works too, my idea with this naming scheme is that we each year we make a new table because what we want in the application may change.  That way we can still access previous years applications.  But also It doesn't seem like being able to access previous years apps is super important lol.

Reply:
Ahh I see. It might be helpful in the future for I guess data reasons. I can start the new application db creation with this change?